ihris-frontend/src/components/ihris/ihris-es-search-term.vue

```diff
@@ -1,86 +1,109 @@
 <template>
   <label>
     <v-autocomplete
-      v-if="isDropDown && !hideFilters"
-      :loading="loading"
-      :label="label"
-      v-model="value"
-      :items="items"
-      outlined
-      :error-messages="err_messages"
-      :error="error"
-      clearable
-      hide-details
-      small-chips
-      dense
-      multiple
-      @change="updateSearch()"
-      @click:clear="clearSearch()"
-      class="ma-1"
-      item-text="display"
-      item-value="code">
+        v-if="isDropDown && !hideFilters"
+        v-model="value"
+        :error="error"
+        :error-messages="err_messages"
+        :items="items"
+        :label="label"
+        :loading="loading"
+        class="ma-1"
+        clearable
+        dense
+        hide-details
+        item-text="display"
+        item-value="code"
+        multiple
+        outlined
+        small-chips
+        @change="updateSearch()"
+        @click:clear="clearSearch()">
       <template v-slot:prepend-item>
         <v-radio-group
-          v-model="filterType"
-          @change="updateSearch()"
-          row
+            v-model="filterType"
+            row
+            @change="updateSearch()"
         >
           <v-radio
-            label="Include"
-            value="include"
+              label="Include"
```

Comment:
Next time can you set your editor to 2 space indentation to avoid making formatting changes